gulpfile.js

```diff
@@ -104,7 +104,7 @@ var cssTasks = function(filename) {
         'opera 12'
       ]
     })
-    .pipe(minifyCss, {
+    .pipe(cssNano, {
       advanced: false,
```

Comment:
As far as I'm aware advanced: false can be removed. That was an oversight on my behalf.